test/eslint-plugin-next/next-script-for-ga.unit.test.js

```diff
@@ -0,0 +1,129 @@
+const rule = require('@next/eslint-plugin-next/lib/rules/next-script-for-ga')
+
+const RuleTester = require('eslint').RuleTester
+
+const ERROR_MSG =
+  'Use the Script component for loading third party scripts. See: https://nextjs.org/docs/messages/next-script-for-ga.'
```

Comment:
```suggestion
  'Use the `next/script` component for loading third party scripts. See: https://nextjs.org/docs/messages/next-script-for-ga.'
```
Should we call it `next/script` here for clarity?